src/ekobox/cli/runcards.py

```diff
@@ -0,0 +1,35 @@
+"""Subcommand to manage runcards."""
+import logging
+import pathlib
+
+from ..cards import generate_operator as opgen
+from ..cards import generate_theory as thgen
+from . import library as lib
+from .base import command
+
+_logger = logging.getLogger(__name__)
+
+DESTINATION = pathlib.Path.cwd().absolute() / "runcards"
+"""Default destination for generated files"""
+
+option_dest = lib.destination(DESTINATION)
+
+
+@command.group("runcards")
+def subcommand():
+    """Manage EKO runcards."""
+
+
+@subcommand.command("example")
+@option_dest
+def sub_example(destination: pathlib.Path):
```

Comment:
- this looks too complicated - we should just create two files and not an additional folder;
- if you want to support custom location (which is fine, but can even be skipped) take two optional arguments `--theory PATH` / `--operator PATH`
- also you should use the constants defined in the other module and drop the `destination` function (which is even inconsistent by the way)

Reply:
> * if you want to support custom location (which is fine, but can even be skipped) take two optional arguments `--theory PATH` / `--operator PATH`

This is definitely redundant: it makes it more complicate, while you can instead rename them with `mv` after generation.

> * also you should use the constants defined in the other module and drop the `destination` function (which is even inconsistent by the way)

Fine for the constants, I just added them later on. The `destination()` function I implemented too many times, simply here we do not have (yet) many commands that produce outputs, but just two. One is this, and the other one already has a complex enough mixture of inputs and outputs.

In any case, the basic idea is that we support smoothly a basic workflow, providing sensible defaults.
E.g.:
```sh
eko runcards example
mv runcards myoperator
cd myoperator
eko run .
```
it is simple enough and works out of the box.